packages/integrations/image/README.md

```diff
@@ -102,7 +296,7 @@ export default {
   
   Image files in your project's `src` directory can be imported in frontmatter and passed directly to the `<Image />` component. All other properties are optional and will default to the original image file's properties if not provided.
 
-```html
+```jsx
```

Comment:
Any reason we're using `jsx` here instead of `astro`? Is it because we want it to show properly on npm?

Reply:
Just an oversight on my end 😅 I was testing a few formats and forgot to revert it back to `astro` highlighting